spec.bs

```diff
@@ -280,9 +280,16 @@ To <dfn>look up penultimate redemption</dfn> given an [=/origin=] |issuer| and a
 1. If [=redemptionTimes=][(|issuer|,|topLevel|)] [=map/exists=], let |penultimateRedemption| be the |penultimateRedemption| field of the tuple [=redemptionTimes=][(|issuer|,|topLevel|)].
 1. Return |penultimateRedemption|.
 
-To <dfn>look up the key commitments</dfn> for a given [=/origin=] |issuer| and an [=/origin=] |topLevel|, run the following steps:
+An user agent has <dfn>pstKeyCommitments</dfn>, a [=map=] where the keys are an [=/origin=] |issuer|, and the values are a [=map=] |commitment|. Populating |pstKeyCommitments| is [=implementation-defined=].
 
-1. TODO
+To <dfn>look up the key commitments</dfn> for a given [=/origin=] |issuer|, run the following steps:
+
+1. If [=pstKeyCommitments=][|issuer|] does not [=map/exist=], return false.
+1. Let |issuerKeys| be [=pstKeyCommitments=][|issuer|].
+1. Let |issuerVersionKeys| be false.
+1. For each |pstVersion| that the user agent supports for this API in an [=implementation-defined=] order, run the following steps:
```

Comment:
I suppose "that the user agent supports for this API" is another way for us to say that this is implementation-defined. That seems fine to me, but it would be nice to further clarify this for implementers, by describing what `pstVersion` is exactly, either in this sentence and/or as a non-normative note below.